DicomRtImportExport/Logic/vtkSlicerDicomRtReader.cxx

```diff
@@ -2156,6 +2250,81 @@ void vtkSlicerDicomRtReader::vtkInternal::LoadRTStructureSet(DcmDataset* dataset
   delete rtStructureSet;
 }
 
+//----------------------------------------------------------------------------
+void vtkSlicerDicomRtReader::vtkInternal::LoadDoseReferencesFromSequence(DRTDoseReferenceSequence& rtDoseReferenceSequence)
+{
+  if (!rtDoseReferenceSequence.gotoFirstItem().good())
+  {
+    vtkErrorWithObjectMacro(this->External, "LoadDoseReferencesFromSequence: No dose references were found");
+    return;
+  }
+  do
+  {
+    DRTDoseReferenceSequence::Item &doseReferenceItem = rtDoseReferenceSequence.getCurrentItem();
+    if (!doseReferenceItem.isValid())
+    {
+      continue;
+    }
+    DoseReferenceEntry doseReferenceEntry;
+    OFCondition cond;
+
+    Float64 weight = -1.;
+    cond = doseReferenceItem.getConstraintWeight(weight);
+    if (cond.good())
+    {
+      doseReferenceEntry.ConstaintWeight = weight;
+    }
+
+    Sint32 refNumber = -1;
+    cond = doseReferenceItem.getDoseReferenceNumber(refNumber);
+    if (cond.good())
+    {
+      doseReferenceEntry.Number = refNumber;
+    }
+
+    OFString structType;
+    cond = doseReferenceItem.getDoseReferenceStructureType(structType);
+    if (cond.good() && !structType.empty())
+    {
+      doseReferenceEntry.StructureType = structType.c_str();
+    }
+
+    Float64 dose = -1.;
+    cond = doseReferenceItem.getTargetMinimumDose(dose);
+    if (cond.good())
+    {
+      doseReferenceEntry.TargetMinimumDose = dose;
+    }
+    cond = doseReferenceItem.getTargetPrescriptionDose(dose);
+    if (cond.good())
+    {
+      doseReferenceEntry.TargetPrescriptionDose = dose;
+    }
+    cond = doseReferenceItem.getTargetMaximumDose(dose);
+    if (cond.good())
+    {
+      doseReferenceEntry.TargetMaximumDose = dose;
+    }

```

Comment:
This only if StructureType == "TARGET"
else:
doseReferenceEntry.OrganMaximumDose = GetOrganAtRiskMaximumDose(dose)

Reply:
StructureType? Not ReferenceType?

The standard says that StructureType: POINT, VOLUME, COORDINATES, SITE;
                                        ReferenceType: TARGET, ORGAN_AT_RISK.